From cba0a8c7c904c9b1186d6985329980370aaef1e1 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Mon, 26 Nov 2018 13:17:14 +0200
Subject: [PATCH 5198/5242] MA-12957: arm64: dts: imx8qm/qxp mek: Correct
 interrupts for adv7535

commit  4a004884a4f16d698941a39fd0ec5f29bedb10ed from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch fixes the interrupts used by ADV7535. Initial patch
configured the GPIO0 IO00 as IO pin for the DSI_INT, used by ADV7535,
but the correct one is IO01, since IO00 is used by PWM.

Fixes: c2f1eceb5629 ("arm64: dts: imx8qm/qxp mek: Configure interrupts
for adv7535")
Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi  |    4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index d5f1472..a280783 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -591,7 +591,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI0_GPIO0_01_MIPI_DSI0_GPIO0_IO01	0x00000020
 			>;
 		};
 
@@ -599,7 +599,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI1_GPIO0_01_MIPI_DSI1_GPIO0_IO01	0x00000020
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
index bcd79ae5..d89109a 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
@@ -456,7 +456,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc6000020
 				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI0_GPIO0_01_MIPI_DSI0_GPIO0_IO01	0x00000020
 			>;
 		};
 
@@ -470,7 +470,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc6000020
 				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
+				SC_P_MIPI_DSI1_GPIO0_01_MIPI_DSI1_GPIO0_IO01	0x00000020
 			>;
 		};
 
-- 
1.7.9.5

